utils/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -73,6 +73,10 @@ class OrtAuthenticator(private val original: Authenticator? = null) : Authentica
         }
     }
 
+    // TODO: Add support for ".authinfo" files (which use the same syntax as .netrc files) once Git.kt does not call the
+    //       Git CLI anymore which only supports ".netrc" (and "_netrc") files.
```

Comment:
Would it make sense to read `_netrc` only if `isWindows == true`?
...e.g. in order to prevent reading such file on other OSes?

Reply:
I was briefly considering this, but I didn't see a risk in unconditionally supporting `_netrc` files in exchange for slightly simpler code.